.github/workflows/test_develop.yml

```
@@ -14,7 +14,7 @@ jobs:
   trigger_tests:
     strategy:
       matrix:
-        branch: [develop, release/cycle-3.118, release/cycle-3.119]  # List other branches here
+        branch: [develop, release/cycle-3.118, release/cycle-3.119, release/cycle-3.120]  # List other branches here
```

you can remove 3.118 since we have 3.119 on prod today
```suggestion
        branch: [develop, release/cycle-3.119, release/cycle-3.120]  # List other branches here
```